server/test/services/export/JsonExporterServiceTest.java

```diff
@@ -890,17 +917,46 @@ public void export_whenFileUploadQuestionIsAnswered_valueIsInResponse() {
         exporter.export(
             fakeProgram.getProgramDefinition(),
             IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
-            SubmittedApplicationFilter.EMPTY);
+            SubmittedApplicationFilter.EMPTY,
+            true);
+    ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
+
+    resultAsserter.assertJsonAtApplicationPath(
+        ".applicant_file",
+        """
+{
+  "file_urls" : [ "%s/admin/applicant-files/test-file-key-1", "%s/admin/applicant-files/test-file-key-2" ],
+  "question_type" : "FILE_UPLOAD"
+}"""
```

Comment:
Did the formatter push this over because the line was so long? Otherwise, can we fix the indentation?

Reply:
Yeah looks like the formatter is doing this